### ISSUE TITLE
Metadata - use CURIEs for contributors

### ISSUE THREAD

**Issue by mbaudis** [ga4gh-schemablocks/blocks-json-schema#6]
In the {S}[B] call from 2019-07-15 

* @ianfore suggested the use of CURIEs (preferably ORCID) for identifying contributors/supporters of the individual blocks
* @MKonopko suggested to align w/ the ResearcherID team

This should be followed up ASAP. In principle, this could either make use of the `ExternalReference` class, or create a class similar to this and `OntologyClass`.

**Comment by Relequestual** [comment]
Should allow the use of ORCID, but not require it.
Should still allow use of name and email.
Some contributors may be in industry only and not in research, and have no interest in establishing an ORCID.

**Comment by mbaudis** [comment]
@Relequestual Yes, not "ORCID only". But not overloading; i.e. "(Name)? & id(s)", not "address".

```
"name": "Michael Baudis"
"uris":
  - "https://api.github.com/users/mbaudis"
  - "mailto:m@baud.is"
  - "https://orcid.org/0000-0002-9903-4248"
  - "http://m.baud.is"
```

But there may be a case for keeping it to one "id", out of a set of selected options.
